src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.vb

```diff
@@ -3406,6 +3406,40 @@ BC30002: Type 'NonExistant' is not defined.
 
             VerifyOperationTreeAndDiagnosticsForTest(Of ObjectCreationExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub IDelegateCreation_SharedAddressOfWithInstanceReceiver()
```

Comment:
>IDelegateCreation_SharedAddressOfWithInstanceReceiver [](start = 19, length = 53)

Please make sure we have tests for accessing a method (static and instance) of off a type

Reply:
Done.

---
In reply to: [149249131](https://github.com/dotnet/roslyn/pull/22972#discussion_r149249131) [](ancestors = 149249131)